LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2017 
+Copyright (c) 2016
```

Comment:
Can you update the copyright to say or include 2017?

Reply:
Done.